### ISSUE TITLE
Add a friendly UI to blocked sites...

### ISSUE THREAD

**Issue by RefinedTheMite** [AdguardTeam/AdGuardHome#6546]
### Prerequisites

- [X] I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer

- [X] I have searched other issues and found no duplicates

- [X] I want to request a feature or enhancement and not ask a question

### The problem

Loading blocked websites that have been blocked on the blocked services settings literally drops the connection... Leaving the user confused if they were unaware of adguard...

### Proposed solution

A friendly UI like what is shown with Adguard parental control stating that the site is blocked makeing look just as visually pleasing to look at like when you attempt to go to a malicious site and get greeted with this...
Something a bit like the image below, just it has been adapted to fit the service blocked.

**Comment by RefinedTheMite** [comment]
https://photos.app.goo.gl/5iWi45sFn4oxUbwg8

**Comment by jumpsmm7** [comment]
> ### Prerequisites
> 
>     * [x]  I have checked the [Wiki](https://github.com/AdguardTeam/AdGuardHome/wiki) and [Discussions](https://github.com/AdguardTeam/AdGuardHome/discussions) and found no answer
> 
>     * [x]  I have searched other issues and found no duplicates
> 
>     * [x]  I want to request a feature or enhancement and not ask a question
> 
> 
> ### The problem
> 
> Loading blocked websites that have been blocked on the blocked services settings literally drops the connection... Leaving the user confused if they were unaware of adguard...
> ### Proposed solution
> 
> A friendly UI like what is shown with Adguard parental control stating that the site is blocked makeing look just as visually pleasing to look at like when you attempt to go to a malicious site and get greeted with this... Something a bit like the image below, just it has been adapted to fit the service blocked.
> ### Alternatives considered and additional information
> 
> _No response_

I could see this as something being useful also to users so they know to alert the admin of the network to false positives.

**Comment by ainar-g** [comment]
Duplicate of #1165.  See [our FAQ](https://github.com/AdguardTeam/AdGuardHome/wiki/FAQ#customblock) for the limitations and the current recommended workaround.
